astropy/wcs/utils.py

```diff
@@ -163,6 +198,19 @@ def _celestial_frame_to_wcs_builtin(frame, projection="TAN"):
         ycoord = "TLAT"
         wcs.wcs.radesys = "ITRS"
         wcs.wcs.dateobs = frame.obstime.utc.isot
+    elif isinstance(frame, BodyBaseCoordinateFrame):
+        if frame.object_name in solar_system_obj_dict.keys():
+            xcoord = solar_system_obj_dict[frame.object_name] + "LN"
+            ycoord = solar_system_obj_dict[frame.object_name] + "LT"
+        else:
+            xcoord = "LON-"
+            ycoord = "LAT-"
+        if frame.obstime is not None:
+            wcs.wcs.dateobs = frame.obstime.utc.isot
+        if issubclass(frame.representation_type, BaseGeodeticRepresentation):
```

Comment:
You mean ITRS, right?
I am confused about the ITRS WCS definition, because of the `TLON` `TLAT` descriptions which I cannot find in the literature... if someone has some references this will help me a lot... Thanks!

Reply:
Indeed, I meant ITRS. I don't know where `TLON`, `TLAT` comes from either... But google gives, e.g., http://tdc-www.harvard.edu/software/wcstools/wcstools.multiwcs.html and https://www.astro.rug.nl/software/kapteyn/allsky.html, so I guess we should just continue to support it.

Reply:
Thanks! I coudn't find those links.